python/BioSimSpace/Align/_lomap/graphgen.py

```diff
@@ -45,14 +45,25 @@
 # MODULE IMPORTS
 # ****************
 
-import networkx as nx
+from BioSimSpace._Utils import _try_import, _have_imported
+
+nx = _try_import("networkx")
+
 import numpy as np
 import subprocess
 import matplotlib.pyplot as plt
 import copy
 from operator import itemgetter
-from rdkit.Chem import Draw
-from rdkit.Chem import AllChem
+
+_rdkit = _try_import("rdkit")
+
+if _have_imported(_rdkit):
+    from rdkit.Chem import Draw
+    from rdkit.Chem import AllChem
+else:
+    Draw = _rdkit
+    AllChem = _rdkit
+
```

Comment:
Using this approach with the bundled LOMAP package might be problematic since we intend to switch to using their conda-package once it is ready. This is a fairly strict requirement of moving to the conda-forge ecosystem, i.e. we can't vendor any other packages. As such, it might be better to simply disable all of the LOMAP functionality within `_align.py` when running on `aarch64`, i.e. we just don't add expose the `generateNetwork` function.